.travis.yml

```diff
@@ -14,7 +14,10 @@ before_script:
   - psql -c 'create database diesel_schema;' -U postgres
 script:
 - |
-  (cd diesel && travis-cargo doc -- --features "postgres sqlite chrono" && echo "docs.diesel.rs" > target/doc/CNAME) &&
+  (cd diesel && travis-cargo doc -- --features "postgres sqlite chrono")
+  if [[ "$TRAVIS_RUST_VERSION" == stable ]]; then
+    (echo "docs.diesel.rs" > diesel/target/doc/CNAME)
+  fi &&
```

Comment:
Hm, I think this whole docs section should be in `after_success` and only run on stable.